templates/check-java.sh.erb

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+# This file is managed by Puppet
+#
+# This file should be identical to 'check-java.sh' in the Jira 7.0.4 standalone
+# tarball, with 2 exceptions: 'grep "java version"' has been changed
+# to 'grep "version"' in order to allow JIRA to run on OpenJDK, and the
```

Comment:
You might think of it as a bit a nitpick, but referring back to the earlier discussion I still think the grep command should be left exactly like the one in the tarball. That way:
- You stick a modified version of the template with the modified grep command in say my_sites/templates/jira/check-java.sh.erb in your module directory (1)
- You pass in an argument as a parameter to this module of ```script_check_java_template => 'my_sites/jira/check-java.sh.erb'```
- Everyone else using this module by default gets the same file as the one that's in the upstream tarball
- Anyone looking at this template sees that we're just taking what is upstream and not doing something which seems weird
- Noone comes along and questions why there is a reference to OpenJDK in the code

Everything else in the PR looks okay.

(1) The my_sites directory in your module directory used to be the 'standard' folder name for the folder containing all of your modified templates. Not sure if there is a more catchy name for it now